libs/graphql/apollo.config.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   client: {
     service: {
       name: 'vega',
-      url: process.env.NX_VEGA_URL,
+      url: process.env.NX_VEGA_URL || 'https://lb.testnet.vega.xyz/query',
```

Comment:
IMO if we want a default we should provide it in a comitted .env rather than just falling back